packages/cli/src/tokens/template.ts

```diff
@@ -0,0 +1,94 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+// import type { ThemeObject } from '@tokens-studio/types';
+// import * as R from 'ramda';
+// import { stringify } from './write';
+
+const DIRNAME: string = import.meta.dirname || __dirname;
+
+const SOURCE_FILES_PATH = path.join(DIRNAME, '../../../../design-tokens');
+const DEFAULT_FILES_PATH = path.join(DIRNAME, './design-tokens/default');
+const TEMPLATE_FILES_PATH = path.join(DIRNAME, './design-tokens/template');
+
+const argsFromToPaths = (path_: string): [string, string] => [
+  path.join(SOURCE_FILES_PATH, path_),
+  path.join(DEFAULT_FILES_PATH, path_),
+];
+
+// const endsWithOneOf = (suffixes: string[], str: string): boolean =>
+//   R.any((suffix: string) => R.endsWith(suffix, str), suffixes);
+
+export const updateTemplates = async () => {
+  // Copy default files
+  await fs.cp(...argsFromToPaths('Figma'), {
+    recursive: true,
+  });
+  await fs.cp(...argsFromToPaths('primitives/globals.json'), {
+    recursive: true,
+  });
+  await fs.cp(...argsFromToPaths('primitives/size/default.json'), {
+    recursive: true,
+  });
+  await fs.cp(...argsFromToPaths('semantic'), {
+    recursive: true,
+  });
+
+  // TODO save for later use
```

Comment:
Not a big fan of keeping code in a comment. I would prefer it to be a function in a separate file, but if you think it will be used VERY soon I guess it's fine.